target/linux/ath79/image/generic.mk

```diff
@@ -193,6 +193,14 @@ define Device/etactica_eg200
 endef
 TARGET_DEVICES += etactica_eg200
 
+define Device/engenius_ews511ap
```

Comment:
Can you move this entry also above etactica_eg200 device?

(also please squash your patches)